File: js/get_campaign_status.js
const url = "<REPLACE_WITH_PROD_URL>/v1/campaigns/1/status";
const token = "<YOUR_TOKEN>";

const fetchData = async () => {
  try {
    const response = await fetch(url, {
      method: "GET",
      headers: {
        Authorization: `Bearer ${token}`,
        "Content-Type": "application/json",
      },
    });

    const data = await response.json();
    console.log(JSON.stringify(data));
  } catch (error) {
    console.log(error);
  }
};

fetchData();

File: python/get_campaign_status.py
import requests

url = "<REPLACE_WITH_PROD_URL>/v1/campaigns/<YOUR_CAMPAIGN_ID>/status"
headers = {
  'Authorization': 'Bearer <YOUR_TOKEN>'
}

response = requests.get(url, headers=headers)
data = response.content.decode("utf-8")
print(data)
